lib/ResizableBox.js

```diff
@@ -7,18 +7,52 @@ import type {Node as ReactNode} from 'react';
 
 type State = {
   width: number, height: number,
-  propsWidth: number, propsHeight: number,
+  propsWidth: number, propsHeight: number
 };
 
-// An example use of Resizable.
 export default class ResizableBox extends React.Component<ResizableProps, State> {
   static propTypes = {
-    height: PropTypes.number,
```

Comment:
- I have seen that `transformScale` prop is not passed to `Resizable` if i use `ResizableBox`. `transformScale` is used in `Resizable` component on line 89. This prop is not mentioned In the documentation. Is this correct?

>If you have a good argument for them...

- PropTypes is a great documentation for every component. 
- It's very important to understand where exactly we are wrong. 
Maybe we are manipulating props like `height`, and we are wrongly passing them to` Resizable`. Through propTypes we immediately understand whether the problem is the value passed to `ResizableBox` or to `Resizable`.

To to avoid circular dependencies i have abstracted flowTypes & propTypes and they are imported from `/lib/propTypes.js`. What do you think?

Reply:
Looks good to me!